src/App.js

```diff
@@ -15,7 +17,7 @@ function App() {
       <Routes>
```

Comment:
If you want your NavBar to only render on the reviews page, you could just make that a child of the AllReviews component